apps/src/aichat/types.ts

```diff
@@ -1,22 +1,35 @@
 import {LevelProperties} from '@cdo/apps/lab2/types';
-import {
-  AiTutorInteractionStatus as AITutorInteractionStatus,
-  PiiTypes as PII,
-} from '@cdo/apps/util/sharedConstants';
-
-// TODO: Update this once https://codedotorg.atlassian.net/browse/CT-471 is resolved
-export type AITutorInteractionStatusType = string;
-
-export {PII, AITutorInteractionStatus};
 
 export type ChatCompletionMessage = {
   id: number;
   role: Role;
   chatMessageText: string;
-  status: AITutorInteractionStatusType;
+  status: AichatInteractionStatus;
   timestamp?: string;
 };
 
+export type ChatContext = {
+  userMessage: string;
+  userId: number;
+  currentLevelId: string | null;
+  scriptId: number | null;
+  channelId: string | undefined;
+};
+
+export enum AichatInteractionStatus {
+  ERROR = 'error',
+  PII_VIOLATION = 'pii_violation',
+  PROFANITY_VIOLATION = 'profanity_violation',
+  OK = 'ok',
+  UNKNOWN = 'unknown',
+}
+
+export enum PiiTypes {
```

Comment:
yes, this is available as a shared constant, but from [this discussion](https://github.com/code-dot-org/code-dot-org/pull/57669#discussion_r1548594055) there seems to be typing issue when working with a shared constant. I can import from shared constants for now and see how it goes when we get to using this constant.

Reply:
Chatted offline with Ben to get more context on this - thanks for sharing prev discussion at https://github.com/code-dot-org/code-dot-org/pull/57669#discussion_r1548185532.
Since @ebeastlake is still unsure whether she will use this on the backend, will leave in shared constants since she will move if she doesn't end up using.